### ISSUE TITLE
Add fields for macro

### ISSUE THREAD

**Issue by awolfe76** [cfpb/hmda-platform#908]
We should add some indication to why the macro edits failed. In other edits this is done by using the fields, but for macro it isn't as straightforward. We'll need more conversation for this and we'll update this issue with details later.

**Comment by schbetsy** [comment]
_This isn't a comment on how this feature_ should _behave, I just wanted to let you know what our current technical capability is._

Once #845 is merged, the `/institutions/<institution>/filings/<period>/submissions/<submissionId>/edits/<type>/<edit>` endpoint will provide information about specific failures of an edit. (Unless this changes in code review,) This is what is returned by default when a macro edit is requested:

```json
{
  "count": 1,
  "total": 1,
  "edit": "Q083",
  "rows": [
    {
      "row": {
        "rowId": ""
      },
      "fields": {
        "Applicant Ethnicity": "error",
        "Property Type": "error",
        "Applicant Race: 1": "error",
        "Applicant Sex": "error",
        "Type of Action Taken": "error"
      }
    }
  ],
  "_links": {
    "self": "?page=1",
    "prev": "?page=1",
    "last": "?page=1",
    "next": "?page=1",
    "first": "?page=1",
    "href": "/institutions/6999998/filings/2017/submissions/3/edits/Q083{rel}"
  }
}
```

That is, it has the same format as the response for other types of edits, but it can't show the value for each field, so it returns `"error"` instead.

So, even without any back-end code changes, the front-end could show _some_ information about a macro edit failure.

**Comment by schbetsy** [comment]
@awolfe76 I'm happy to sit down with you any time this week to discuss how this feature should look.

**Comment by schbetsy** [comment]
Update: @awolfe76 and I discussed this issue. We decided that the description of the edit (which already comes from the API in the `/edits` endpoint) provides enough information about why the edit failed. For now, the front-end will just show that description. If we want to show more information in the future, we'll make another ticket.
